halibot/halibot.py

```diff
@@ -90,13 +133,25 @@ def add_instance(self, name, inst):
 		self.log.info("Instantiated object '" + name + "'")
 
 	def _load_config(self):
+		# Special values expanded inside of configs
+		specials = {
+			'HALDIR': HALDIR
```

Comment:
of Lothlórien!

Reply:
I was thinking `HALPATH` might be better, but you have convinced me otherwise.